packages/taquito-utils/src/taquito-utils.ts

```diff
@@ -63,6 +63,14 @@ export function b58cencode(value: string | Uint8Array, prefix: Uint8Array) {
   return bs58check.encode(Buffer.from(n.buffer));
 }
 
+export function b58cencodeRollupAddress(value: string | Uint8Array) {
```

Comment:
could you add unit tests for b58cencodeRollupAddress, b58decode where the payload is a txr1, encodePubKey where the value is a txr1 (binary format)?